.benchmarks/gobenchdata-checks.yml

```
@@ -0,0 +1,25 @@
+checks:
+    - name: VM checks
+      description: |-
+        It checks performance regressions on VM package.
```

```suggestion
        It checks performance regressions on VM full test suite.
```

or something else, but 'package' is invalid.